docs/source/examples/template.myst

```diff
@@ -3,8 +3,8 @@ jupytext:
   text_representation:
     extension: .md
     format_name: myst
-    format_version: '0.8'
-    jupytext_version: '1.4.1'
+    format_version: 0.13
+    jupytext_version: 1.11.4
```

Comment:
Not specific to this PR, but the amount of churn from different versions is pretty annoying.

Reply:
It was an attempt to regularize, they would not need changed often. Before they were all over the place and I wanted there to be consistency in the behavior/expectations from document to document.